javascript/javascript2/week3/script.js

```diff
@@ -0,0 +1,145 @@
+// 1. Warmup
+// Log out the text Called after 2.5 seconds 2.5 seconds after the script is loaded.
+
+setTimeout((() => console.log("Called after 2.5 seconds")), 2500);
+
+// Create a function that takes 2 parameters: delay and stringToLog. Calling this function should log out the stringToLog after delay seconds. Call the function you have created with some different arguments.
+
+function delayFunction(delay, stringToLog) {
+    return setTimeout((() => console.log(stringToLog)), delay * 1000);  
+}
+
+delayFunction(5, 'This string logged after 5 seconds');
+
+// Create a button in html. When clicking this button, use the function you created in the previous task to log out the text: Called after 5 seconds 5 seconds after the button is clicked.
+
+const btn = document.querySelector('.click');
+btn.addEventListener('click', () => {
+   return setTimeout((() => console.log("Called after 5 seconds")), 5000);
```

Comment:
You do not have to use the `return` keyword here. It will work without it and we do not store the return value anywhere anyway. Also could you maybe use your `delayFunction` here? :)